packages/moonstone/ExpandableItem/ExpandableItem.js

```diff
@@ -332,7 +332,7 @@ const ExpandableItemBase = kind({
 			// When disabled, don't attach an event
 			if (!disabled) {
 				if (open) {
-					onClose(ev);
+					onClose && onClose(ev);
```

Comment:
Should this guard `onOpen`, too?

Reply:
@webOS101 , Currently, there's no problem.

Reply:
There would be, though.  Since it's not required, please add the same guard below.

Reply:
Done